src/app/fsat/explore-opportunities/explore-opportunities-form/explore-opportunities-form.component.ts

```diff
@@ -110,7 +110,7 @@ export class ExploreOpportunitiesFormComponent implements OnInit {
   }
 
   initForms() {
-    if (!this.fsat.isVFD || !this.fsat.modifications[this.exploreModIndex].exploreOppsShowVfd) {
+    if (!this.fsat.isVFD && !this.fsat.modifications[this.exploreModIndex].exploreOppsShowVfd) {
```

Comment:
I don't think this is the correct logic.

Use an installed version of the app to select isVFD and then close the app and run this branch. The isVFD isn't selected.